### ISSUE TITLE
Colortag elements' types don't reflect behavior

### ISSUE THREAD

**Issue by etanttila** [apluslms/mooc-jutut#63]
The colortag buttons in the feedback response panels for tagging feedback with feedback tags are of the type `span`. However, they are clickable and clicking on them performs an action (tagging the feedback), so they behave as buttons. Therefore, it would be better that the elements are actually buttons rather than spans. (Otherwise, they should at least have the role of a button.) Currently, they are not focusable, which is unaccessible for keyboard navigation. If they were buttons, they would automatically be focusable.
![image](https://github.com/apluslms/mooc-jutut/assets/50318434/be2f9ca6-bbd7-4456-ad32-a3c5768c154b)

On the other hand, in the _Manage feedback tags_ -view (titled _List of feedback tags_), if one clicks on a tag in the list, there is a click handler, which attempts to tag it to a feedback, which fails, resulting in a warning. However, this should not have a click handler (and should be a span).
![image](https://github.com/apluslms/mooc-jutut/assets/50318434/db782c47-355a-46c2-b053-3c2146856a81)

If one changes the earlier mentioned button-behaving colortags to buttons, in the file feedback.js, the clickhandler `ajax_set_tag_state` could be added to only buttons with the class colortag rather than all elements with the class, which would resolve the problem of click handlers (and possibly styling?) being applied to spans, that should not behave in a button-like manner.
